### PR TITLE
[FEATURE] Allow the tt_content_defValues to be set for a container

### PR DESCRIPTION
**Pull request by nigelmann**
With the `setTtContentDefValues` method we can directly set for a given container what it's default values are.

The CType is set by default

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@ This is an example to create a 2 column container. The code snippet goes into a
 | `setSaveAndCloseInNewContentElementWizard` | saveAndClose for new content element wizard (v10 only) | `bool $saveAndCloseInNewContentElementWizard` | `true` |
 | `setRegisterInNewContentElementWizard` |register in new content element wizard | `bool $registerInNewContentElementWizard` | `true` |
 | `setGroup` | Custom Group (used as optgroup for CType select (v10 only), and as tab in New Content Element Wiazrd). If empty "container" is used as tab and no optgroup in CType is used. | `string $group` | `'container'` |
+| `setTtContentDefValues` | Default values for the newContentElement.wizardItems | `array $ttContentDefValues` | `[]` |
```
Comment (bmack) [inline README.md:80]: shouldn't we use a more speaking option name? Something like "defaultValues" ?

**Comment by achimfritz** [comment]
hm, i would prefer to configure tt_content_defValues for the CE in the new content element wizard in your site-extension PageTS, e.g.

`mod.wizards.newContentElement.wizardItems.<my-group>.elements.<my-CType>.tt_content_defValues.header = bar`

because to not bloat the EXT:container Code with Configurations that are seldom required, but can be easy set by TYPO3 "Standard-Configuration"

(and btw. i have expected to set the tt_content_defValues with an "key => value" array
`->setTtContentDefValues(['header' => 'foo'])`
but a string is required
`->setTtContentDefValues(['header = foo'])`
)

**Comment by nigelmann** [comment]
Hi @achimfritz 

Thanks for your feedback.

I understand your point and for my usage setting the defaut values via TsConfig works fine.

The advantage for me was that it allowed to group all the configuration related to the container elements I created in one place.

For the array of strings I did it this way to avoid to then have to recreate those strings in the Registery.

Let me know if you think this feature is worth adding to the extension and I will make necessary changes.

Cheers,  
Simon

**Comment by achimfritz** [comment]
Hi @nigelmann , our integrators likes your feature, if you make the changes, we will merge. Thanks for MR.

**Comment by nigelmann** [comment]
Hi @achimfritz, just adapted according to feedback, let me know if anything more needs to be changed

Thanks for work !

**Comment by achimfritz** [comment]
@nigelmann thanks for your effort

**Pull request merged**